src/components/multiStepBar/MultiStepBar.tsx

```diff
@@ -0,0 +1,67 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import {IClassName} from '../../utils/ClassNameUtils';
+import {ITooltipProps, Tooltip} from '../tooltip/Tooltip';
+
+export type IMultiStepState = 'to-do' | 'doing' | 'done' | 'error' | 'warning';
```

Comment:
what about using an enum like so:  
  
```
enum MultiStepState {
    toDo = 'to-do',
    doing = 'doing',
    error = 'error',
    warning = 'warning'
}

interface IStep {
    state: MultiStepState;
    text?: string;
    tooltip?: ITooltipProps;
}  
```  
you could reuse your enum properties everywhere you need to pass a MultiStepState